posthog/settings.py

```diff
@@ -157,7 +157,12 @@ def print_warning(warning_lines: Sequence[str]):
     _clickhouse_http_protocol = "https://"
     _clickhouse_http_port = "8443"
 
-CLICKHOUSE_HTTP_URL = _clickhouse_http_protocol + CLICKHOUSE_HOST + ":" + _clickhouse_http_port + "/"
+CLICKHOUSE_HTTP_URL = f"{_clickhouse_http_protocol}{CLICKHOUSE_HOST}:{_clickhouse_http_port}/"
+
+_clickhouse_hosts = get_from_env("CLICKHOUSE_MIGRATION_HOSTS", CLICKHOUSE_HOST).split(",")
+CLICKHOUSE_MIGRATION_HOSTS_HTTP_URLS = [
+    f"{_clickhouse_http_protocol}{host}:{_clickhouse_http_port}/" for host in _clickhouse_hosts
+]
```

Comment:
Maybe consider making `f"{_clickhouse_http_protocol}{host}:{_clickhouse_http_port}/"` a function with `host` param, so it's easier to read/edit later?

Reply:
In general I agree, but I'd avoid introducing functions in settings.py due to convention. Given there are only two instances of this + they're close by I don't think it's that bad yet. :)

Reply:
totally fine to leave it as is.

n00b q: what convention is that as I do see other functions in this file?